web/server.js

```diff
@@ -87,28 +79,26 @@ module.exports = function(options) {
         fontSrc: [
           '\'self\'',
           'https://fonts.gstatic.com'
-        ]
+        ],
+        generateNonces: false
```

Comment:
apparently blankie now auto-generates a few nonces that are injected into the csp. No idea what those nonces are used for but I disabled it.

Reply:
yeah, these broke donate too when I updated blankie. The nonces are supposed to let you allow inline scripts via CSP, I think.